pkg/server/config.go

```diff
@@ -331,7 +331,10 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 		apiHandler = WithRequestIdentity(apiHandler)
 		apiHandler = authorization.WithDeepSubjectAccessReview(apiHandler)
 
+		authorizerWithoutAudit := genericConfig.Authorization.Authorizer
+		genericConfig.Authorization.Authorizer = authorization.EnableAuditLogging(genericConfig.Authorization.Authorizer)
 		apiHandler = genericapiserver.DefaultBuildHandlerChainFromAuthz(apiHandler, genericConfig)
+		genericConfig.Authorization.Authorizer = authorizerWithoutAudit
```

Comment:
isn't `authorizerWithoutAudit` what we pass to  `WithHomeWorkspaces` ? (line 342)

Reply:
if I don't pass it there, I don't get audits at all.

Reply:
let me retest 🤔

Reply:
yeah, i believe something changed and `WithHomeWorkspaces` now hosts regular workspaces as well after the refactoring? (not sure)

Reply:
my audit logs are empty with what is in `main` without this PR.

Reply:
> yeah, i believe something changed and WithHomeWorkspaces now hosts regular workspaces as well after the refactoring? (not sure)

yeah, I think that after the refactoring the vw server project `workspaces` onto `clusterworkspaces`.

Reply:
shouldn't we be passing an authoriser decorated with `authorization.EnableAuditLogging` ?

Reply:
`genericConfig.Authorization.Authorizer = authorization.EnableAuditLogging(genericConfig.Authorization.Authorizer)`

Reply:
don't we pass `genericConfig.Authorization.Authorizer` which is overwritten in line `337` (`authorizerWithoutAudit`) to `WithHomeWorkspaces`?

Reply:
he, yes, that's confusing 😆 You're right, but the audit enabled handler is given to the wrapped api handler. I'll comment to make it clearer 👍